agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -231,6 +233,11 @@ function install_k8s_agent {
         rm /tmp/sysdig-agent-daemonset-v2.yaml.bak
     fi
 
+    if [ ! -z "$SYSDIG_REPO" ]; then
+        sed -i.bak -e 's|\(.*image: icr.io/ext/sysdig/agent\)|        image: '$SYSDIG_REPO'|g' /tmp/sysdig-agent-daemonset-v2.yaml
```

Comment:
if the indentation changes in the upstream yaml, this sed will break

Reply:
the use of embedded single quotes is unnecessary. Please use double quotes around the expression and no quotes inside

Reply:
I think that this sed can be tighter
`"s|\(.*\)image:\(.*\)|/1image: $SYSDIG_REPO|g"`